### PR TITLE
Update data explorer

### PR DESCRIPTION
**Pull request by emeeks**
* Update Semiotic version to allow for large scatterplots with interactivity
* Update controls to attractive blueprint controls (buttons and select)

### CODE REVIEW

File: packages/presentational-components/src/components/source.md [hunk 1]
```diff
@@ -82,7 +82,6 @@ class Editor extends React.Component {
         <textarea
           onChange={evt => {
             this.setState({ value: evt.target.value });
-            console.log(evt.target.value);
           }}
```
Comment (rgbkrk) [inline packages/presentational-components/src/components/source.md:86]: Was this too noisy in this demo code section?

File: packages/presentational-components/src/vendor/blueprint-select.css.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
+import css from 'styled-jsx/css';
+
+export default css.global`/*!
```
Comment (rgbkrk) [inline packages/presentational-components/src/vendor/blueprint-select.css.js]: If you rebase, this is CSS is now available via

```
import { blueprintSelectCSS } from "@nteract/styled-blueprintjsx";
```

File: packages/transform-dataresource/package.json [hunk 3]
```diff
@@ -44,8 +44,8 @@
     "react-hot-loader": "^4.1.2",
     "react-table": "^6.8.6",
     "react-table-hoc-fixed-columns": "1.0.2",
-    "semiotic": "^1.14.4",
     "styled-jsx": "^3.1.0",
```
Comment (rgbkrk) [inline packages/transform-dataresource/package.json:47]: include this as a dependency

```suggestion
    "styled-jsx": "^3.1.0",
    "@nteract/styled-blueprintjsx": "^1.0.0",
```

File: packages/transform-dataresource/src/VizControls.js [hunk 4]
```diff
@@ -1,8 +1,112 @@
 import * as React from "react";
+import { Select } from "@blueprintjs/select";
+import { Button, ButtonGroup, MenuItem, Code } from "@blueprintjs/core";
+
+import {
+  blueprintCSS,
+  blueprintSelectCSS
+} from "../../styled-blueprintjsx/src/index";
```
Comment (rgbkrk) [inline packages/transform-dataresource/src/VizControls.js]:
```suggestion
} from "@nteract/styled-blueprintjsx";
```

File: packages/transform-dataresource/src/VizControls.js [hunk 5]
```diff
@@ -259,40 +383,53 @@ export default ({
...
+                </Button>
+              ))}
+            </ButtonGroup>
           </div>
         )}
       </div>
       <style jsx>{chartUIStyle}</style>
       <style jsx>{buttonGroupStyle}</style>
+      <style jsx>{blueprintCSS}</style>
+      <style jsx>{blueprintSelectCSS}</style>
```
Comment (rgbkrk) [inline packages/transform-dataresource/src/VizControls.js:432]: 👍

File: packages/presentational-components/src/vendor/blueprint-select.css.js (hunk 2 above)
Comment (rgbkrk) [reply packages/presentational-components/src/vendor/blueprint-select.css.js]: I see you're using it below so you can `git rm` this file now.

File: packages/presentational-components/src/components/source.md (hunk 1 above)
Comment (emeeks) [reply packages/presentational-components/src/components/source.md:86]: No I should have left this in, I didn't notice that it was in the demo code. I was just trying to clean up all my console.logs and must have cut this one unintentionally.

**Pull request merged**